app/__init__.py

```diff
@@ -20,7 +20,6 @@
 if app.config.get('TIMESYNC_URL') == 'http://timesync.example.com/':
     print 'ERROR: Configuration option "TIMESYNC_URL" not set'
     print '       Please set "TIMESYNC_URL" to the URL of your TimeSync server'
-    sys.exit(1)
```

Comment:
Hmm... By removing this the program will not exit when the URL is set to the example. This is good for our tests, but usually we do want the program to exit. Instead, you should add an additional condition to the if statement making sure the program is not in test mode.